styles/styles.css

```
@@ -347,10 +521,32 @@ body {
   font-size: 18px;
   line-height: 21px;
   font-weight: 400;
-  color: #333333;
+  width: 292px;
   border: 2px solid #e5e5e5;
-  padding: 14px 23px;
   border-radius: 4px;
+  padding: 14px 20px;
+  margin-left: 130px;
+  background-image: url("../images/select-arrow.svg");
+  background-repeat: no-repeat;
+  background-position: right 16px center;
```

```suggestion
  background-position: 95%;
```
я бы попробовал как-то так с процентами поиграться тут